README.md

```diff
@@ -50,7 +50,7 @@ Source: https://github.com/cncf/artwork/tree/master/projects/kubernetes/certifie
 
 | Release | Manifest | Kubernetes Version |
 | --- | --- | --- |
-| 18 | [v1-22-eks-18](https://distro.eks.amazonaws.com/kubernetes-1-22/kubernetes-1-22-eks-18.yaml) | [v1.22.17](https://github.com/kubernetes/kubernetes/release/tag/v1.22.17) |
+| 19 | [v1-22-eks-19](https://distro.eks.amazonaws.com/kubernetes-1-22/kubernetes-1-22-eks-19.yaml) | [v1.22.17](https://github.com/kubernetes/kubernetes/release/tag/v1.22.17) |
```

Comment:
@danbudris so the version number wouldn't change for this one?

Reply:
Our release number does because we cut another rlease, but the underlying k8s version didn't change. It's still .17: https://github.com/aws/eks-distro/blob/main/projects/kubernetes/kubernetes/1-22/GIT_TAG